docs/content/contrib-development.md

```diff
@@ -81,7 +81,7 @@ git commit -s
 git push origin somefeature
```

Comment:
Thanks for fixing these! As you've probably guessed, they're remnants from moving a markdown file that was only accessible via github into the website 😳